src/components/sections/DmEditSidebar.vue

```diff
@@ -22,7 +44,22 @@ import DmSectionCard from './DmSectionCard.vue';
 
 export default {
   data: () => ({
+    search: "",
     show: false,
+    showFilter: false,
+    selectedLevel: [],
+    selectedShift: '',
+    levelOptions: [
+      { value: 0, name: "Optativo" },
+      { value: 1, name: "1" },
+      { value: 2, name: "2" },
+      { value: 3, name: "3" },
+      { value: 4, name: "4" },
+      { value: 5, name: "5" },
+      { value: 6, name: "6" },
+      { value: 7, name: "7" },
+      { value: 8, name: "8" },
```

Comment:
Filter the levels from the incomming sections. Never hardcode things that can generalize